pkg/metrics/metrics.go

```diff
@@ -11,28 +9,36 @@ const (
 
 	sharesSubsystem = "openshift_csi_share"
 
-	mount          = "mount"
-	mountCountName = sharesSubsystem + separator + mount + separator + "total"
+	mount                 = "mount"
+	mountCountName        = sharesSubsystem + separator + mount + separator + "requests_total"
+	mountFailureCountName = sharesSubsystem + separator + mount + separator + "failures_total"
 
 	MetricsPort = 6000
 )
 
 var (
-	mountCounter = createMountCounter()
+	mountCounter, failedMountCounter = createMountCounter()
```

Comment:
Style nit - pluralize
```suggestion
	mountCounter, failedMountCounter = createMountCounters()
```